### PR TITLE
Deprecated BUILD_UNIT_TEST + removed legacy files + updated README file (with pip installation)

### PR DESCRIPTION
**Pull request by cmastalli**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,18 +27,24 @@ The source code is released under the [BSD 3-Clause license](LICENSE).
 [![GitHub repo size](https://img.shields.io/github/repo-size/loco-3d/crocoddyl)](https://img.shields.io/github/repo-size/loco-3d/crocoddyl)
 [![conde version](https://img.shields.io/conda/vn/conda-forge/crocoddyl.svg)](https://anaconda.org/conda-forge/crocoddyl)
 [![conde install](https://anaconda.org/conda-forge/crocoddyl/badges/installer/conda.svg)](https://anaconda.org/conda-forge/crocoddyl)
+[![PyPI](https://badge.fury.io/py/crocoddyl.svg)](https://pypi.org/project/crocoddyl/)
 [![conde download](https://anaconda.org/conda-forge/crocoddyl/badges/downloads.svg)](https://anaconda.org/conda-forge/crocoddyl)
```
Comment (proyan) [inline README.md:31]: I didn't realize it before. This should be "conda download". COuld you please fix it here?

File: README.md [hunk 2]
```diff
@@ -27,18 +27,24 @@ The source code is released under the [BSD 3-Clause license](LICENSE).
...
 [![contributors](https://img.shields.io/github/contributors/loco-3d/crocoddyl)](https://github.com/loco-3d/crocoddyl/graphs/contributors)
 [![GitHub Release Date](https://img.shields.io/github/release-date/loco-3d/crocoddyl)](https://img.shields.io/github/release-date/loco-3d/crocoddyl)
 [![GitHub last commit](https://img.shields.io/github/last-commit/loco-3d/crocoddyl)](https://img.shields.io/github/last-commit/loco-3d/crocoddyl)
 
 If you want to follow the current developments, you can directly refer to the [devel branch](https://github.com/loco-3d/cddp/tree/devel).
-If you want to directly dive into **Crocoddyl**, only one single line is sufficient (assuming you have Conda):
+If you want to directly dive into **Crocoddyl**, only one single line is sufficient
+
+<p align="center">
+pip install --user crocoddyl
```
Comment (proyan) [inline README.md:40]:
```suggestion
pip3 install --user crocoddyl  # for python 3 users
or 
pip install --user crocoddyl  # for deprecated python 2 support
```

**Review by proyan — COMMENTED** [review]
Thanks for the PR Carlos. I have made some minor suggestions.

Could you also please make a PR to the master branch with the changes in the README.md?

**Comment by proyan** [comment]
Also, the master branch remains C++ 98/11/14/17/20, however, since the merging of the residual PR, the devel would no longer support C++ 98 standard. So we should make that change in the README as well.

File: README.md (hunk 1 above)
Comment (cmastalli) [reply README.md:31]: done

File: README.md (hunk 2 above)
Comment (cmastalli) [reply README.md:40]: done

**Comment by cmastalli** [comment]
@proyan -- I realized that some of the Python-based unit-tests were using deprecated functions. I have to clean up a bit this. Additionally, I am defining `nthreads=1` for Python-based models as Crocoddyl doesn't yet support multithreading with Python models.

**Comment by cmastalli** [comment]
I will merge this PR as gitlab-ci passed and the CI for testing the code format in Travis passed (this is all we need to be sure that everything is alright!)

**Comment by jcarpent** [comment]
Pip is only for Linux. I would keep conda as default one.

File: README.md [hunk 3]
```diff
@@ -23,22 +23,32 @@ The source code is released under the [BSD 3-Clause license](LICENSE).
...
-[![GitHub Release Date](https://img.shields.io/github/release-date/loco-3d/crocoddyl)](https://img.shields.io/github/release-date/loco-3d/crocoddyl)
 [![GitHub last commit](https://img.shields.io/github/last-commit/loco-3d/crocoddyl)](https://img.shields.io/github/last-commit/loco-3d/crocoddyl)
+[![GitHub repo size](https://img.shields.io/github/repo-size/loco-3d/crocoddyl)](https://img.shields.io/github/repo-size/loco-3d/crocoddyl)
 
 If you want to follow the current developments, you can directly refer to the [devel branch](https://github.com/loco-3d/cddp/tree/devel).
-If you want to directly dive into **Crocoddyl**, only one single line is sufficient (assuming you have Conda):
+If you want to directly dive into **Crocoddyl**, only one single line is sufficient
+
+<p align="center">
+pip3 install --user crocoddyl  # for python 3 users
```
Comment (jcarpent) [inline README.md:40]: Don’t make the distinguo between version 2 and 3.

Comment (cmastalli) [reply README.md:40]: > Pip is only for Linux. I would keep conda as default one.

I could reorder this, i.e., adding first conda and later pip. 

> Don’t make the distinguo between version 2 and 3.

Not sure what do you mean here? Would you prefer to keep one instruction using `pip` only?


I could do both actions in a PR that I am preparing now

Comment (cmastalli) [reply README.md:40]: done. See ongoing PR for more details

**Pull request merged**